test/resque_inline_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class IgnoreExceptionsTest < Minitest::Test
+  def setup
+    Resque.stubs(:redis)
+    Resque.inline = true
+  end
+
+  def teardown
+    Resque.inline = false
+    Resque.unstub(:redis)
+  end
+
+  def test_ignore_exceptions
+    GoodJob.stubs(:perform).raises(CustomException)
```

Comment:
Thanks, that's good to know! I'm always afraid I'll dishearten someone when I've absolutely no intention of doing so :)